src/components/method-common.js

```diff
@@ -47,3 +47,14 @@ export const convertToRawUrl = (methodPath, methodVersion, methodSource) => {
     }
   )
 }
+
+export const getMethodVersionName = url => {
```

Comment:
Is it possible to get the version tag from Github itself using the Github url?

Reply:
That should be possible, and the `else` statement here should cover getting the version tag from a github.com link. Are you referencing a different kind of github url?

Reply:
Sorry waht I meant was ping Github or its APIs to obtain the version or branch name using the Github url itself rather than doing the String manipulation.

Reply:
Ah yes, I tried to look for an API that could get this for us, but it seemed for all of the potential endpoints, I needed the branch name anyway. I'll give the documentation a once over one more time for good measure.

Reply:
Oh I see. And just to make sure, the position of version tag / branch name is always consistent? Hence there is no risk with doing string splicing this way?

Reply:
Not that I was able to find. I tried methods from different repositories, as well as trying files at different depths in their respective directory trees.

Reply:
I see. I guess it might be worth adding different url tests (apart from the 3 you already have) for this method to ensure we don't extract the wrong version / branch name for the url.